### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by natalierjackson**
Code Navigation team requested a new category name

### CODE REVIEW

**Comment by Natka666** [comment]
what can I do ? and neither i cant  restore my old account

**Comment by mathurs26** [comment]
Previous Page

**Comment by Doctor1187** [comment]
Access

**Comment by Mcquistion89** [comment]
Whatsbthos

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Natka666** [comment]
what is now ? i cant run this license?

**Comment by Natka666** [comment]
?

**Review by michellemerrill — CHANGES_REQUESTED** [review]
@natalierjackson To make it consistent, please take out the word "feedback"

**Comment by natalierjackson** [comment]
@michellemerrill looking for second view

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ In this repository, you can find the [official GitHub public feedback discussion
 | **Feedback Category** | **About the Product** 	|
 |---	|---	|
 | 🚢  [Actions and Packages](https://github.com/github/feedback/discussions/categories/actions-and-packages-feedback) 	| [GitHub Actions](https://github.com/features/actions) and [GitHub Packages](https://github.com/features/packages) |
-| 🔎  [Code Search & Navigation Feedback](https://github.com/github/feedback/discussions/categories/code-search-and-navigation-feedback) 	| [Code Search & Navigation Feedback](https://cs.github.com/about) 	|
+| 🔎  [Code Search & Navigatio](https://github.com/github/feedback/discussions/categories/code-search-and-navigation-feedback) 	| [Code Search & Navigation](https://cs.github.com/about) 	|
```
Comment (lucalves) [inline README.md:8]: I suggest fixing `Navigatio` to `Navigation`.

**Comment by Natka666** [comment]
fine but it is my coding..

**Comment by Natka666** [comment]
ok. Sorry ;)

**Review by Jigorey — COMMENTED** [review]
Logging issue while gaming

**Review by michellemerrill — COMMENTED** [review]
@natalierjackson Correct spelling on Navigation then LGTM!

**Review by michellemerrill — APPROVED** [review]
LGTM

**Pull request merged**